app/src/androidTest/java/edu/uwstout/p2pchat/ExampleInstrumentedTest.java

```diff
@@ -36,4 +36,12 @@ public void useAppContext()
 
         assertEquals("com.uwstout.p2pchat", appContext.getPackageName());
     }
+    @Test
```

Comment:
adding these tests back will destroy instrumented tests. Also, the fact that you haven't deleted these tells me that you didn't do any instrumented tests.